server/server.py

```diff
@@ -167,21 +167,25 @@ def post_disable_rule(self):
         response = BytesIO()
         response.write(str(success).encode('utf-8'))
         self.wfile.write(response.getvalue())
-    
-    def post_add_rule(self):
+
+    def post_rewrite_rule(self):
```

Comment:
Here you already had a thought that the backend endpoint can be unified into one `post_rewrite_rule`. Then how about we unify the endpoint into one `/saveRule`? And the frontend does not have to differentiate them either, right?

Reply:
I just wanted to know when debugging if a rule was coming from "add a rule" or "edit a rule". But I will change it. It seems like it won't matter too much since they share most functions.